games/tests/test_integration.py

```diff
@@ -0,0 +1,123 @@
+from django.test import Client
+from django.urls import reverse
+
+from players.factories import PlayerFactory
+from testing import BaseTestCase
+from testing.utils import css_select_get_attributes, css_select_get_text
+
+
+class IntegrationTestCase(BaseTestCase):
```

Comment:
This will end up being a class to make integration tests easier. It (for the moment), will assume that there are two players. Each has their own client, so we can check the state of the game for each of them pretty easily. At the moment, it can load the game and play an action. It can also detect the number of actions/buys/money you have. And there is an easy tool to get the list of cards in your hand. There isn't anything in there for adhoc turn stuff yet, but that should be pretty easy to hook in.